catalogapp/src/deb/resources/etc/georchestra/catalogapp/js/GEOR_custom.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Sample geOrchestra catlogapp config file
+ *
+ * Instructions: 
+ * uncomment lines you wish to modify and 
+ * modify the corresponding values to suit your needs.
+ */
+
+Ext.namespace("GEOR");
+
+GEOR.custom = {
+    
+    /**
+     * Constant: HEADER_HEIGHT
+     * Integer value representing the header height, as set in the shared maven filters
+     * Defaults to 90
+     */
+    HEADER_HEIGHT: 90,
```

Comment:
We should find a way to get this value from the server side var headerHeight (homecontroller ?)